zipline/data/minute_bars.py

```diff
@@ -520,6 +525,10 @@ def __init__(self, rootdir):
 
         self._first_trading_day = metadata.first_trading_day
         self._minute_index = metadata.minute_index
+
+        self._minute_value_index = \
```

Comment:
I would recommend this instead for clarity, `self._minute_index.values.astype('datetime64[m]).astype(int)`.